web/src/components/tasks/business-formation/billing/getCost.test.ts

```diff
@@ -29,7 +18,36 @@ describe("getCost", () => {
     });
   }
 
+  describe("official formation document", () => {
+    const overriddenCostIds: FormationLegalType[] = ["nonprofit"];
+    const defaultCostIds = allFormationLegalTypes.filter((it) => !overriddenCostIds.includes(it));
+
+    for (const legalStructureId of defaultCostIds) {
```

Comment:
[dust] Is there a reason we are using the for loop here instead of the it.each pattern? I think the for loop is bit more readable in this case but was wondering if there were other things taken into consideration. 

`it.each([...defaultCostIds])("...", (legalStructureId) => {})`

Reply:
I was just following the existing pattern of the file already... happy to change if we prefer it.each. Thoughts?

Reply:
I don't have a preference but I think we should chat as a dev team on whether we have preferred choice and try to have it reflect throughout our code for uniformity sake or is having both patterns okay? I was in a similar situation for the signer and checkbox error feature where the test files had `for` loops and I refactored them to follow the `it` pattern but either would have been fine.